sematic/ui/packages/common/src/typeViz/vizMapping.ts

```diff
@@ -45,6 +46,7 @@ const meta: Array<[string, RenderDetails]> = [
     ["datetime.datetime", { value: DatetimeValueView }],
     ["enum.Enum", { value: EnumValueView }],
     ["plotly.graph_objs._figure.Figure", { value: PlotlyFigureValueView, nested: PlotlyFigureExpandedView }],
+    ["sematic.types.types.prompt_response.PromptResponse", { value: PromptResponseCollapsedView, nested: PromptResponseExpandedView}],
```

Comment:
would you mind having the `sematic.types.types` stay together ? :D

Reply:
done